pkg/deviceclaimingserver/enddevices/ttjsv2/ttjs.go

```diff
@@ -109,6 +110,8 @@ var (
 	errDeviceNotClaimed     = errors.DefineNotFound("device_not_claimed", "device with EUI `{dev_eui}` not claimed")
 	errDeviceAccessDenied   = errors.DefinePermissionDenied("device_access_denied", "access to device with `{dev_eui}` denied: device is already claimed or the owner token is invalid") //nolint:lll
 	errUnauthenticated      = errors.DefineUnauthenticated("unauthenticated", "unauthenticated")
+	errUnclaimDevice        = errors.Define("unclaim_device", "could not unclaim device with EUI `{dev_eui}`", "message")
+	errUnclaimDevices       = errors.Define("unclaim_devices", "could not unclaim devices")
```

Comment:
We don't write "failed to" and "could not" etc in error messages. Errors imply failure already.

Reply:
We do, but ok. I've removed these
<img width="568" alt="Screenshot 2023-08-16 at 09 19 35" src="https://github.com/TheThingsNetwork/lorawan-stack/assets/13186113/79ca328f-2ff0-46f6-9c3c-44fec08462c2">

Reply:
Yes these are mostly old and wrong too.